skymp5-server/cpp/server_guest_lib/PapyrusActor.cpp

```diff
@@ -3,6 +3,40 @@
 #include "MpActor.h"
 #include "MpFormGameObject.h"
 
+namespace {
+std::string ToLowerCase(std::string str)
```

Comment:
Please for case insensitive string comparison use [this](https://github.com/skyrim-multiplayer/skymp/blob/main/skymp5-server/cpp/papyrus_vm_lib/CIString.h) or [this](https://github.com/skyrim-multiplayer/skymp/blob/3b92b19e903540bc773f8a04b39da9049e6b73d7/skymp5-server/cpp/papyrus_vm_lib/Utils.cpp) (clickable)